include/z64actor.h

```diff
@@ -232,55 +252,90 @@ typedef struct {
     /* 0x0450 */ Vec3f      unk_450;
     /* 0x045C */ char       unk_45C[0x00E];
     /* 0x046A */ u16        unk_46A;
-    /* 0x046C */ char       unk_46C[0x6E];
+    /* 0x046C */ char       unk_46C[0x06E];
     /* 0x04DA */ s16        unk_4DA;
-    /* 0x04DC */ char       unk_4DC[0x188];
+    /* 0x04DC */ char       unk_4DC[0x008];
+    /* 0x04E4 */ Collider   unk_4E4; // TODO determine type
+    /* 0x04FC */ char       unk_4FC[0x068];
+    /* 0x0564 */ Collider   unk_564; // TODO determine type
+    /* 0x057C */ char       unk_57C[0x07C];
+    /* 0x05F8 */ u8         unk_5F8;
+    /* 0x05F9 */ char       unk_5F9[0x06B];
     /* 0x0664 */ Actor*     unk_664;
     /* 0x0668 */ char       unk_668[0x004];
     /* 0x066C */ s32        unk_66C;
-    /* 0x0670 */ char       unk_670[0x00C];
+    /* 0x0670 */ u32        unk_670; // (sword?) effect related
+    /* 0x0674 */ ActorFunc  actionFunc;
```

Comment:
i guess we can start doing the PlayerActionFunc typedef for link too.
and thanks for the comments on the other variables, def helps